assets/css/common/type.scss

```diff
@@ -64,6 +64,20 @@ blockquote {
 
 }
 
+div.highlight {
+  margin: 0 0 $gutter 0;
+  padding: $gutterHalf;
+  background-color: #e1e8e8;
```

Comment:
We may move this colour code as well as some others using `#e1e8e8` to `assets/css/common/variables.scss` in future but it should be done in a separate card I think.